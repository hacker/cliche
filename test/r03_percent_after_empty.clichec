#include <iostream>
int main() {<%output>
Test for broken handling of '^% ' line following the empty line.
Like this:

% std::cout << 1 << std::endl;

</%output>}
