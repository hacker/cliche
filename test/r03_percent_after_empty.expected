
Test for broken handling of '^% ' line following the empty line.
Like this:

1

